pennylane_lightning/core/lightning_base.py

```diff
@@ -50,7 +50,7 @@ class LightningBase(QubitDevice):
             OpenMP.
     """
 
-    pennylane_requires = ">=0.38"
+    pennylane_requires = ">=0.40"
```

Comment:
You've updated the referenced version. If so, don't forget to update the changelog.

Reply:
Reverted